app/src/main/java/org/greenstand/android/TreeTracker/activities/SplashFragment.kt

```diff
@@ -24,6 +26,7 @@ class SplashFragment : Fragment() {
 
     private val user: User by inject()
     private val preferencesMigrator: PreferencesMigrator by inject()
+    private val languageSwitcher: LanguageSwitcher by inject()
```

Comment:
I don't see languageSwitcher used in this class, am I missing something?

Reply:
Good catch